torch_xla_py/xla_model.py

```diff
@@ -309,6 +309,27 @@ def log_fn(step_result):
   return log_fn
 
 
+def check_view_sharing(obj):
+  tensors = set()
+  aliases = dict()
+
+  def check_object(obj):
+    if is_xla_tensor(obj):
```

Comment:
Do we not handle `nn.Module` objects from `torch.save(model)`?

Reply:
The new code should (see `__dict_` enumeration) ...

Reply:
https://github.com/pytorch/xla/pull/1040/files#diff-7b3dae7482135fbad6523ab4d0d9ed48R104